library/std/src/os/windows/io/raw.rs

```diff
@@ -11,7 +11,6 @@ use crate::os::windows::io::{OwnedHandle, OwnedSocket};
 use crate::os::windows::raw;
 use crate::ptr;
 use crate::sys;
-use crate::sys::c;
```

Comment:
I'm not exactly sure why we're churning this part. I'm in favor of cleanup but I'd like to understand the directionality here.

Reply:
Because otherwise rustdoc complains. Literally the only reason.

Reply:
To explain that further, it's an "unresolved import" when compiling the std docs from non-windows because their sys directory likely doesn't have a `c` module.

Reply:
Ahh, I see.